movieapp/src/services/constants.ts

```diff
@@ -7,6 +7,7 @@ export const CONSTANTS = {
   ADD_FORM_TITLE: 'add_form_title',
   LABEL_TITLE: 'label_title',
   LABEL_DESCRIPTION: 'label_description',
+  LABEL_DESCRIPTION_FULL: 'label_description_full',
```

Comment:
bad practice to call enums/objects in plural `CONSTANTS` and it's super common 

it's Localization enum/object

Reply:
fixed